docs/source/configuration_file.rst

```diff
@@ -146,6 +146,11 @@ The resulting section should look as follows ::
                          pattern: '.*\.mov'
```

Comment:
would you change this line to `pattern: '*.mov'` please?

Reply:
Would the change work in the current setting? I haven't implemented my template update on Box yet.

Reply:
Will be done later ...